### ISSUE TITLE
UI is blurry on HiDPI screens on MacOS (retina) since 1.2.x

### ISSUE THREAD

**Issue by antwan** [IJHack/QtPass#355]
Steps to reproduce: Open the app, it's blurry.

Don't know if the dmg was built with a wrong version of QT, but it looks like a massive regression as soon as you open it on a hi-dpi machine (most of them are now).

**Comment by lukedirtwalker** [comment]
Hi @Antwan86 can you please add a screenshot to see the problem? Did version 1.2.0 not have this problem?

**Comment by Antwan86** [comment]
Before (1.1.x)
<img width="453" alt="1 1 6" src="https://user-images.githubusercontent.com/3027728/37589732-0dd33b3e-2b5e-11e8-9553-f8c4753bf097.png">



After (1.2.x)
![1 2 1](https://user-images.githubusercontent.com/3027728/37589746-120db922-2b5e-11e8-9a9d-4e74a3442fed.png)

**Comment by lukedirtwalker** [comment]
Hm 1.1.6 dmg file includes a Info.plist file and in 1.2.1 it is missing. So I'm pretty sure this causes the problem. From this page: http://doc.qt.io/qt-5/highdpi.html I would guess the missing part is the `NSPrincipalClass`.

I'm not sure where you would need to put the Info.plist file (no Mac user), but judging from 1.1.6 its content should be something like this:
```<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
        <key>CFBundleDisplayName</key>
        <string>QtPass 1.2.1</string>
        <key>CFBundleExecutable</key>
        <string>QtPass</string>
        <key>CFBundleGetInfoString</key>
        <string>1.2.1</string>
        <key>CFBundleIconFile</key>
        <string>icon.icns</string>
        <key>CFBundleIdentifier</key>
        <string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
        <key>CFBundlePackageType</key>
        <string>APPL</string>
        <key>CFBundleShortVersionString</key>
        <string>1.2</string>
        <key>CFBundleSignature</key>
        <string>????</string>
        <key>NOTE</key>
        <string>QtPass is a multi-platform GUI for pass</string>
        <key>NSHumanReadableCopyright</key>
        <string>Copyright © 2014-2018 IJhack
This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.</string>
        <key>NSPrincipalClass</key>
        <string>NSApplication</string>
</dict>
</plist>
```

**Comment by cgrinds** [comment]
Thanks, @lukedirtwalker. I can confirm that fixes the bluriness when added to `/Applications/QtPass.app/Contents/Info.plist`

**Comment by pheraph** [comment]
Icons are still blurry on macOS using v1.3.2. Please reopen the issue.

**Comment by annejan** [comment]
Fixed in upcoming 1.3.3 release
